compiler/rustc_passes/src/stability.rs

```diff
@@ -411,9 +445,15 @@ impl<'a, 'tcx> Visitor<'tcx> for Annotator<'a, 'tcx> {
     }
 
     fn visit_trait_item(&mut self, ti: &'tcx hir::TraitItem<'tcx>) {
+        let fn_sig = match ti.kind {
+            hir::TraitItemKind::Fn(ref fn_sig, _) => Some(fn_sig),
+            _ => None,
+        };
```

Comment:
This can be simplified using an `if let` expression
```suggestion
        let fn_sig = if let hir::TraitItemKind::Fn(ref fn_sig, _) = ti.kind {
            Some(fn_sig)
        } else {
            None
        };
```

Reply:
I do prefer `match` usually. Is it something "required" for this to go through?
I feel like it's more concise and it's clearer

Reply:
Right, I guess it's just a matter of taste, don't mind me.